### ISSUE TITLE
[Frontend] Add confirmation message on currency modification

### ISSUE THREAD

**Issue by FelipePSoares** [FelipePSoares/EconoFlow#255]
Show a alert message when user modify currency on user profile, to advice him that just the symbol will change but the values will keep the same, we will not convert the values. 

Message: "The values ​​will **not** be changed, only the currency symbol displayed on the screens will be changed."

**Comment by ahmedhalac** [comment]
Hi @FelipePSoares. Do u want this message appear every time the currency changes, or only after the user saves the edited form?

In which form this message should be displayed. Can it be within toaster?

**Comment by FelipePSoares** [comment]
Hi @ahmedhalac ,

I was planning this message appear before save the form ask for a confirmation for the user. Like when you delete or archive something.

What do you think?

**Comment by ahmedhalac** [comment]
So as I understood correctly, this should be displayed like modal when user hits save button?

**Comment by ahmedhalac** [comment]
Hi @FelipePSoares. Just a quick question here. What options should be there on modal? 

Cancel/Ok?

**Comment by FelipePSoares** [comment]
Hi @ahmedhalac , I think that "Cancel/Confirm" or "Cancel/Save", but you can choose what you think that is better.

**Comment by FelipePSoares** [comment]
Hey @ahmedhalac ,

**Comment by FelipePSoares** [comment]
Ok @ahmedhalac , thank you for everything.

**Comment by FelipePSoares** [comment]
@saksham-malhotra-27 you ask me about another issue to start, this could be a good one if you want. 

Do you prefer some backend or frontend stuff?

**Comment by saksham-malhotra-27** [comment]
Alright ! Will do this one

**Comment by saksham-malhotra-27** [comment]
Hey @FelipePSoares ! Actually, I'm more of a backend guy, but I'm not familiar with .NET, that's why I'm focusing more on frontend tasks for this project. However, I'm comfortable with tech stack involving like Express or Hono, and I also have some experience with Go, just not .NET yet. That said, I’ve been exploring the project to get a better understanding of how .NET works.
